### ISSUE TITLE
HA Lovelace setpoint differs from Toons setpoint

### ISSUE THREAD

**Issue by guidokieboom** [cyberjunky/home-assistant-toon_climate#12]
Hi Cyberjunky,

I'm very pleased with your work on Toon Climate. I'm not a programmer; just basic level experience in using HA and basic understand of yaml. After setting up my rooted Toon, I noticed a that when I adjust the temperature setpoint on the Toon screen, this does not update the Lovelace setpoint. See example below (I've set Lovelace setpoint at 17.5 and later on the Toon screen at 18.5. The 18.5 is not updated in Lovelace.

<img width="732" alt="Screenshot 2020-10-04 at 12 06 30" src="https://user-images.githubusercontent.com/65387179/95012656-ade0e900-063a-11eb-998c-1a61f2f2f99d.png">

Second question relates to the error in Lovelace "custom element does not exist". I took this from the example in the information page on HACS. Would you know the cause?

Cheers, Guido

**Comment by guidokieboom** [comment]
After hitting "Flush Firewall" on the Toon device, synching Toon and iOS devices vv started to work.
Question w.r.t. simple-thermostat remains.

**Comment by cyberjunky** [comment]
If you configured your Lovelace via the GUI, did you configure and entry 'bronnen/sources' to contain:

/hacsfiles/simple-thermostat/simple-thermostat.js
as javascrip module?

**Comment by guidokieboom** [comment]
Thanks Ron for getting on this. I did not configure as I'm not aware this should be done (for the other Lovelace card I did not do anything). I would have no idea where to start as I don't have any experience with programming. I tried the following in LoveLace GUI, but error still same. Also I don't see a folder with hacsfiles on my RPi.
<img width="1142" alt="Screenshot 2020-10-06 at 21 36 02" src="https://user-images.githubusercontent.com/65387179/95251562-06bba780-081c-11eb-8735-2c5395b202c2.png">
